src/pages/RFOX/components/Claim/ClaimRow.tsx

```diff
@@ -69,49 +95,57 @@ export const ClaimRow: FC<ClaimRowProps> = ({
           : 'RFOX.tooltips.unstakePendingCooldown',
         { cooldownPeriodHuman },
       )}
+      isDisabled={displayClaimButton}
     >
       <Flex
-        as={Button}
         justifyContent={'space-between'}
-        mt={2}
         align='center'
-        variant='unstyled'
-        p={8}
+        p={2}
         borderRadius='md'
+        height='auto'
         width='100%'
-        onClick={handleClaimClick}
-        isDisabled={status !== ClaimStatus.Available}
-        _hover={hoverProps}
+        {...parentProps}
       >
         <Flex>
-          <Box mr={4}>
+          <Flex alignItems='center' mr={4}>
             <AssetIconWithBadge assetId={foxAssetId}>
               <TransactionTypeIcon type={TransferType.Receive} />
             </AssetIconWithBadge>
-          </Box>
+          </Flex>
           <Box mr={4}>
             <RawText fontSize='sm' color='gray.400' align={'start'}>
-              {translate('RFOX.claim', { assetSymbol: stakingAssetSymbol })}
+              {actionDescription
+                ? actionDescription
+                : translate('RFOX.claim', { assetSymbol: stakingAssetSymbol })}
```

Comment:
`stakingAssetSymbol` can be undefined here
<img width="797" alt="image" src="https://github.com/shapeshift/web/assets/17035424/e6d13f47-038e-490b-9ac2-bdd888d0b10c">

```suggestion
                : translate('RFOX.claim', { assetSymbol: stakingAssetSymbol ?? '' })}
```

Ideally, we would guard against this though

Reply:
Done in [683d93f](https://github.com/shapeshift/web/pull/7098/commits/683d93fbc32c413ab11ded7c3829d3d96e3720c9)